### PR TITLE
Discord logic updates as of 12/20/24

### PR DESCRIPTION
**Pull request by Rooby-Roo**
I looked for comma issues manually this time and my linter didn't pick anything up so this is a perfect PR please dont test or review just merge

### CODE REVIEW

File: worlds/animal_well/region_data.py [hunk 1]
```diff
@@ -442,15 +442,17 @@ class AWData(NamedTuple):
...
+                                     [iname.wheel_hop, iname.matchbox]],
                    loc_type=LocType.candle),
         lname.candle_bear_event:
-            AWData(AWType.location, [[iname.bubble, iname.matchbox], [iname.disc, iname.matchbox]],
+            AWData(AWType.location, [[iname.bubble, iname.matchbox], [iname.disc, iname.matchbox],
+                                     [iname.wheel_hop, iname.matchbox]],
                    event=iname.event_candle_bear),
         lname.fruit_49:
-            AWData(AWType.location, [[iname.bubble], [iname.disc]], loc_type=LocType.fruit),
+            AWData(AWType.location, [[iname.bubble], [iname.disc], [iname.matchbox]], loc_type=LocType.fruit),
```
Comment (ScipioWright) [inline worlds/animal_well/region_data.py]:
```suggestion
            AWData(AWType.location, [[iname.bubble], [iname.disc], [iname.wheel_hop]], loc_type=LocType.fruit),
```

File: worlds/animal_well/region_data.py [hunk 2]
```diff
@@ -442,15 +442,17 @@ class AWData(NamedTuple):
...
         lname.fruit_50:
-            AWData(AWType.location, [[iname.bubble], [iname.disc]], loc_type=LocType.fruit),
+            AWData(AWType.location, [[iname.bubble], [iname.disc], [iname.matchbox]], loc_type=LocType.fruit),
```
Comment (ScipioWright) [inline worlds/animal_well/region_data.py]:
```suggestion
            AWData(AWType.location, [[iname.bubble], [iname.disc], [iname.wheel_hop]], loc_type=LocType.fruit),
```
I'm assuming this was the intent, yeah?

File: worlds/animal_well/region_data.py [hunk 3]
```diff
@@ -490,13 +492,15 @@ class AWData(NamedTuple):
             AWData(AWType.location, loc_type=LocType.fruit),
         rname.bear_ladder_after_chameleon:
             AWData(AWType.region),  # just press a button
-        # just throw the ball into the hole, it'll hit the yellow button like 99% of the time
+        rname.bear_slink_room:
+            AWData(AWType.region),  # just go up
```
Comment (ScipioWright) [inline worlds/animal_well/region_data.py]:
```suggestion
        rname.bear_slink_room:
            AWData(AWType.region, [[iname.bubble], [iname.disc_hop], [iname.wheel_hop]]),  # just go up
```
There's way too much vertical distance needed to just jump up this -- you need a bubble, a kinda tricky disc hop, or a wheel hop

**Review by ScipioWright — CHANGES_REQUESTED** [review]

File: worlds/animal_well/region_data.py (hunk 2 above)
Comment (Rooby-Roo) [reply worlds/animal_well/region_data.py]: Zero idea how this happened lmao

File: worlds/animal_well/region_data.py (hunk 3 above)
Comment (Rooby-Roo) [reply worlds/animal_well/region_data.py]: I think this is up there for "the most wrong I've been about something that doesn't matter".

Comment (ScipioWright) [reply worlds/animal_well/region_data.py]: idk I think that goes to the matchbox up above lmao

**Pull request merged**